tests/playlistFavorites.spec.js

```diff
@@ -126,6 +126,8 @@ describe('Test it can get all favorites for a playlist with playlists_favorites
 describe('Test the delete playlist endpoint', () => {
   beforeEach(async () => {
     await database.raw('truncate table playlists cascade');
+    await database.raw('truncate table playlists_favorites cascade');
```

Comment:
I added these truncate/cascades to help me get past the foreign key/duplicate errors we were experiencing earlier

Reply:
Good call, I had to add that to a couple tests as well